json-data-on-the-web CI check timing out

The notebook `applications/json-data-on-the-web.ipynb` appears to be timing out when the CI checks run. Just thought I'd flag that here in case it's a problem (might just be a flaky test/notebook, though)

```
=================================== FAILURES ===================================
___________________ applications/json-data-on-the-web.ipynb ____________________
[gw0] linux -- Python 3.8.10 /usr/share/miniconda3/envs/dask-examples/bin/python3.8
A cell timed out while it was being executed, after 300 seconds.
The message was: Cell execution timed out.
Here is a preview of the cell contents:
-------------------
events.pluck('spec').frequencies(sort=True).take(20)
-------------------
----------------------------- Captured stderr call -----------------------------
[IPKernelApp] WARNING | Parent appears to have exited, shutting down.
------------------------------ Captured log call -------------------------------
ERROR    traitlets:client.py:582 Timeout waiting for execute reply (300s).
=========================== short test summary info ============================
FAILED applications/json-data-on-the-web.ipynb::
=================== 1 failed, 37 passed in 593.61s (0:09:53) ===================
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci-build.yaml
+++ b/.github/workflows/ci-build.yaml
@@ -31,7 +31,7 @@ jobs:
             --nbmake \
             --overwrite \
             --forked \
-            --ignore machine-learning/torch-prediction.ipynb
+            --ignore machine-learning/torch-prediction.ipynb \
             --ignore applications/json-data-on-the-web.ipynb
 
       - name: Build

